covid_graphs/Dockerfile

```diff
@@ -6,6 +6,5 @@ RUN apt update \
     && apt install -y python3-pip python3-numpy python3-pandas \
     && rm -rf /tmp/* /var/lib/apt/lists/* /root/.cache/*
 
-COPY requirements.txt .
-
-RUN pip3 install -r requirements.txt
+COPY . .
+RUN pip3 install .
```

Comment:
I did not manage to make this work locally. @lukipuki how is this file used?

Reply:
`COPY . .` looks wrong (it should be a no-op).
Strange that pip3 install doesn't work. I'll try it myself.

Reply:
The following works
```
FROM ubuntu:19.10

ENV DEBIAN_FRONTEND=noninteractive
RUN echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf
RUN apt update \
    && apt install -y python3-pip python3-numpy python3-pandas \
    && rm -rf /tmp/* /var/lib/apt/lists/* /root/.cache/*

COPY . /covid19_graphs
RUN pip3 install /covid19_graphs
RUN rm -rf /covid19_graphs
```